tests/network/samba/samba_adcli.pm

```diff
@@ -51,6 +51,42 @@ sub samba_sssd_install {
     script_run('dig srv _ldap._tcp.geeko.com');
 }
 
+sub update_password {
+    # Array @params contains tuples with [$params-for-adcli, $adcli-output-substring]
+    my @params = (['', 'WBC_ERR_AUTH_ERROR'], ['--add-samba-data', 'succeeded']);
```

Comment:
~~why the empty param?~~

Could you write a better explanation of the purpose of this array? took me a moment to understand it

Reply:
Took ma also time to understand it, explanation would help. But I don't mind because it would be much faster if it would be another command I know the params and how it bahaves. :)

Reply:
Hope the comment makes better sense now. The reason I run the commands with this somehow cryptic array is that, in case that the commands fail, I need to run the same steps each time (like the `adcli join`).